conda_smithy/tests/test_lint_recipe.py

```diff
@@ -148,6 +162,8 @@ def test_cli_success(self):
                 fh.write(textwrap.dedent("""
                     package:
                         name: 'test_package'
+                    build:
+                        number: 0
```

Comment:
Not really. See https://travis-ci.org/conda-forge/conda-smithy/builds/122514255#L358. (Travis-CI is green though.)